avocado/plugins/multiplex.py

```diff
@@ -35,23 +34,28 @@ def __init__(self, *args, **kwargs):
 
     def configure(self, parser):
         parser = super(Multiplex, self).configure(parser)
+        parser.add_argument("--summary", type=int, help="Verbosity of "
```

Comment:
I think we need to differentiate between 2 things:

1) Have the output or not (at a standard verbosity level)
2) The level of verbosity (which indeed has to be flexible to accommodate custom plugins)

So, if we expect the variants plugins to provide two *different* kinds of information, such as:

1) variants listing
2) variants summary

We should have options such as:

1) `--variants-list`
2) `--variants-summary`

When given, they will print output at a standard verbosity level.  To change that, other parameters options could be given:

1) `--variants-list=info`
2) `--variants-summary=debug`

If I were to code this, I'd align `info`, `debug`, etc with `logging.INFO` (numeric decimal 10), `logging.DEBUG` (numeric decimal 20), which would probably not surprise many people, and still allow custom plugins to define meanings for things such as value 15, 19, 25, etc.